### DISCUSSION TITLE
加载阿里云open总是失败，提示如下：failed get objs: failed to list objs: Post "https://openapi.aliyundrive.com/adrive/v1.0/openFile/list": dial tcp ************53:443: connect: connection refused

### DISCUSSION THREAD

**Discussion by xiaohanleng-lc** [AlistGo/alist#4346] [Q&A]
版本3.16.3
日志内容：
stderr: [31mERRO[0m[2023-05-08 15:30:32] failed get enabled storages: Post "https://openapi.aliyundrive.com/adrive/v1.0/user/getDriveInfo": dial tcp ************53:443: connect: connection refused
stderr: failed init storage
stderr: github.com/alist-org/alist/v3/internal/op.initStorage
stderr: 	/app/internal/op/storage.go:99
stderr: github.com/alist-org/alist/v3/internal/op.LoadStorage
stderr: 	/app/internal/op/storage.go:80
stderr: github.com/alist-org/alist/v3/internal/bootstrap.LoadStorages.func1
stderr: 	/app/internal/bootstrap/storage.go:20
stderr: runtime.goexit
stderr: 	/usr/lib/go/src/runtime/asm_arm64.s:1172 
stderr: 2023/05/08 15:47:26.900760 ERROR RESTY Post "https://openapi.aliyundrive.com/adrive/v1.0/openFile/list": dial tcp ************53:443:connect: connection refused, Attempt 1
stderr: 2023/05/08 15:47:27.050622 ERROR RESTY Post "https://openapi.aliyundrive.com/adrive/v1.0/openFile/list": dial tcp ************53:443:connect: connection refused, Attempt 2
stderr: 2023/05/08 15:47:27.176984 ERROR RESTY Post "https://openapi.aliyundrive.com/adrive/v1.0/openFile/list": dial tcp ************53:443: connect: connection refused, Attempt 3
stderr: 2023/05/08 15:47:27.436294 ERROR RESTY Post "https://openapi.aliyundrive.com/adrive/v1.0/openFile/list": dial tcp ************53:443:connect: connection refused, Attempt 4

**Comment by anwen-anyi** [comment]
不应该考虑一下你自己网络的问题吗？ 阿里云盘海外IP不一定能用

**Reply by xiaohanleng-lc** [reply]
这恰恰就是最神奇的事情，本地ip不行，非得海ip才行，我真的会谢

**Reply by anwen-anyi** [reply]
加到host 试试
- 47.94.245.197 api.aliyundrive.com
- 47.94.245.197 auth.aliyundrive.com
- 121.89.3.30 cn-beijing-data.aliyundrive.net

**Reply by xiaohanleng-lc** [reply]
好的，谢谢！

**Reply by xiaohanleng-lc** [reply]
还是不行，也是搞不懂了，而且必须用clash的全局代理才行

**Reply by zorgzola** [reply]
遇到一样问题。我部署了两个，一个内、一个外。外的正常，内的和楼主一样的报错。

**Comment by im-dashan** [comment]
我用的新加坡的公网服务器，支持ipv4和ipv6，但是还是会出现这个错误，这要怎么解决呢

**Comment by tkzzzzzz6** [comment]
我也有这样的情况,但是重启一下服务器alist又可以正常工作了
